tests/test_tinytext.py

```diff
@@ -11,3 +11,16 @@ def test_something() -> None:
 
     # Assert
     assert tiny == "ᶦᶰᵗᵒ ᵗᶦᶰᶦᵉʳ ᵗᵉˣᵗ"
+
+
+def test_for_digits() -> None:
+    # Arrange
+    numbers = "".join(
+        str(character) for character in range(9, -1, -1)
+    )  # numbers = 9876543210
```

Comment:
Shorter, easier to read, and more explicit:
```suggestion
    numbers = "9876543210"
```

Reply:
Sure, updated with its direct value as suggested